src/components/ComponentPages/Registration/UI/email.tsx

```diff
@@ -0,0 +1,102 @@
+import { Typography, Form, Input, Button } from "antd";
+import {
+  CloseCircleOutlined,
+  ArrowRightOutlined,
+  RedoOutlined,
+} from "@ant-design/icons";
+
+import styles from "./Registration.module.scss";
+
+import messages from "../../../../messages";
+
+const { Title, Text } = Typography;
+
+const EmailConfirmation = ({
+  form,
+  onFinish,
+  isModal,
+  closeModal,
+  isOTP,
+  onResendClick,
+  isResend,
+}) => (
+  <section className={styles.signup_wrapper + " " + styles.textCenter}>
+    <Form
+      form={form}
+      name="registration"
+      onFinish={onFinish}
+      layout="vertical"
+      scrollToFirstError
+      validateTrigger={messages.formValidationTypes()}
+    >
+      <Title level={2} className={styles.titles}>
+        {isOTP ? "One Time Verification Code" : "User Email Confirmation"}
+      </Title>
+      {isModal && (
+        <Button
+          shape="circle"
+          type="link"
+          className={styles.close_btn}
+          onClick={closeModal}
+          icon={<CloseCircleOutlined />}
+        />
+      )}
+      <div className={styles.section_one}>
+        <Text type="danger" className={styles.otpNote}>
+          {isOTP
+            ? `Note : Verification code has been sent to your registered email address.`
+            : `Note : Your email address is not returned from social account. You
```

Comment:
Message must be come from the common message file

Reply:
done